tests/functional/basic/Report.lua

```diff
@@ -155,6 +154,21 @@ return {
 	end,
 	addMatrix = function(unitTest)
```

Comment:
This tests must me removed, as there is no funcion with this name.

Reply:
I removed all test from the new functions.